processor/schemaprocessor/README.md

```diff
@@ -0,0 +1,24 @@
+# Schema Transformer Processor
+
+🚧 _Currently under development, subject to change_ 🚧
+
+Supported Pipelines: traces, metrics, logs
+
+The _Schema Processor_ is used to convert existing telemetry data or signals to a version of the semantic convention defined as part of the configuration. 
+The latest schema translation file will be bundled as part of the collector build to avoiding having to fetch it at runtime. 
```

Comment:
The "latest" changes over time, specification publishes new versions and the "latest" schema release in Otel may happen after the particular bundle of the Collector is released. What happens in that case?
I think the Collector should be able to fetch the schema file on demand. Bundled "latest schema available at the release time may be an optimization to speed up Collector startup (but I am not totally sure it is necessary).

Reply:
I think it is reasonable to be not in complete lockstep, having the context on your idea of how you expect the processor to work, missing a schema translation file could be resolved at runtime.

However, this idea is a rather round about way and there is currently nothing set up to do the automation.

I should be clear that I read this idea from the spec for it and potentially considering moving this automation til after the ability to transform these signals is complete.